BTCPayServer/Payments/PayJoin/PayJoinEndpointController.cs

```diff
@@ -405,21 +409,35 @@ public async Task<IActionResult> Submit(string cryptoCode, string invoice)
 
             foreach (var availableUtxo in availableUtxos)
             {
+                if (currentTry >= maxTries)
+                    return Array.Empty<UTXO>();
                 //we can only check against our input as we dont know the value of the rest.
```

Comment:
This is no longer true since you now know the value of the inputs through core or the psbt